orchestrationSpecs/packages/migration-workflow-templates/tests/__snapshots__/outputMatch.test.ts.snap

```diff
@@ -3571,6 +3638,18 @@ echo $SHOULD_FINALIZE",
           ],
           "image": "{{inputs.parameters.imageMigrationConsoleLocation}}",
           "pullPolicy": "{{inputs.parameters.imageMigrationConsolePullPolicy}}",
+          "resources": {
+            "limits": {
+              "cpu": "4000m",
+              "ephemeral-storage": "20G",
+              "memory": "4G",
+            },
+            "requests": {
+              "cpu": "1000m",
+              "ephemeral-storage": "20G",
+              "memory": "4G",
+            },
+          },
```

Comment:
Can you break up the test cases so that there's a separate snapshot file for each template?  It's very challenging to find which template these are for in a PR diff.  Sorry, I should have done that when I set the test up!